_conferences/droids-4-droids-colombia-2018.md

```diff
@@ -0,0 +1,12 @@
+---
+name: "Droids4Droids"
+website: http://droids4droids.co/
+location: Medellin, ANT, COLOMBIA
```

Comment:
Is it traditional to capitalize these completely?

Also what is the ANT here? Forgive my lack of culture...